articles/Articles.tsx

```diff
@@ -0,0 +1,183 @@
+    date:"Dec 22, 2022",
+    text1:"COVID-19 Was a Top Cause of",
+    text2:"Death in 2020 and 2021, Even",
+    text3:"For Younger People",
+    Covid19:"Covid-19"
+  },
+
+  {
+    id:2,
+    Imaage:require("../assets/articlesImages/Image.png"),
+    date:"Dec 22, 2022",
+    text1:"Study Finds Being 'Hangry' is",
+    text2:"a Real Thing",
+    Covid19:"Health"
+  },
+
+  {
+    id:3,
+    Imaage:require("../assets/articlesImages/Image (1).png"),
+    date:"Dec 22, 2022",
+    text1:"Study Finds Being 'Hangry' is",
+    text2:"a Real Thing",
+    Covid19:"Health"
+  },
+
+  {
+    id:4,
+    Imaage:require("../assets/articlesImages/Image (1).png"),
+    date:"Dec 22, 2022",
+    text1:"Study Finds Being 'Hangry' is",
+    text2:"a Real Thing",
+    Covid19:"Health"
+  }
+]
+
+
+export default function Article() {
+  const navigation = useNavigation();
+  return (
+    <SafeAreaView style={{marginTop:50}}>
+      <ScrollView showsHorizontalScrollIndicator={false}>
+      <View style={{display:"flex",flexDirection:"row",justifyContent:"space-between"}}>
+          <View style={{display:"flex",flexDirection:"row",gap:15,padding:15}}>
+            <Image source={require("../assets/articlesImages/Vector (2).png")}/>
+            <Text style={{fontSize:24,color:"#212121",fontWeight:"bold"}}>Articles</Text>
+          </View>
+            
+          <View style={{display:"flex",flexDirection:"row",gap:10,padding:15}}>
+            <TouchableOpacity><Image style={{position:"relative",marginTop:10,padding:8}}source={require("../assets/articlesImages/Search.png")}/></TouchableOpacity>
+            <TouchableOpacity onPress={()=>navigation.navigate("BookMarkedArticle"as never)}><Image style={{position:"relative",marginTop:10,padding:8}}source={require("../assets/articlesImages/Bookmark.png")}/></TouchableOpacity>
+          </View>
+    
+      </View>
+
+      <View style={{display:"flex",flexDirection:"row",justifyContent:"space-between",padding:15}}>
+        <Text style={{fontSize:20,color:"#212121",fontWeight:"bold"}}>Trending</Text>
+        <TouchableOpacity onPress={()=>navigation.navigate("SeeAllArticles" as never)}><Text style={{color:"#246BFD",fontSize:16}}>See All</Text></TouchableOpacity>
+      </View>
+
+
+      <ScrollView horizontal={true} showsHorizontalScrollIndicator={false}>
+        {articles.map(article=>(
+          <View key={article.id}style={{display:"flex",flexDirection:"row",marginLeft:15}}>
+          <View style={{display:"flex",flexDirection:"column"}}>
+            <TouchableOpacity onPress={()=>navigation.navigate("ArticlesDetails"as never)}><Image style={{height:140,width:220,borderRadius:20}}source={article.image}/></TouchableOpacity>
+            <View style={{flexWrap:"wrap",paddingTop:15}}>
+            <Text style={{fontSize:18,color:"#212121",fontWeight:"bold",width:230}}>{article.title}</Text>
+            <Text style={{fontSize:18,color:"#212121",fontWeight:"bold",width:230}}>{article.title2}</Text>
```

Comment:
Is it necessary to split the title in 2 parts? The title should be one and it can wrap if the article card had a maximum width.